### DISCUSSION TITLE
This project is a godsend

### DISCUSSION THREAD

**Discussion by j1mmie** [wovalle/fireorm#321] [General]
```
 ██████████ ██                         ██                                 ██
░░░░░██░░░ ░██                        ░██        ██   ██                 ░██
    ░██    ░██       ██████   ███████ ░██  ██   ░░██ ██   ██████  ██   ██░██
    ░██    ░██████  ░░░░░░██ ░░██░░░██░██ ██     ░░███   ██░░░░██░██  ░██░██
    ░██    ░██░░░██  ███████  ░██  ░██░████       ░██   ░██   ░██░██  ░██░██
    ░██    ░██  ░██ ██░░░░██  ░██  ░██░██░██      ██    ░██   ░██░██  ░██░░ 
    ░██    ░██  ░██░░████████ ███  ░██░██░░██    ██     ░░██████ ░░██████ ██
    ░░     ░░   ░░  ░░░░░░░░ ░░░   ░░ ░░  ░░    ░░       ░░░░░░   ░░░░░░ ░░ 

```

Open source is often a thankless job, so I wanted to make sure you see something *other* than a bug report or feature request in your feed.

After spending way-too-much time wrestling with the Firebase Admin SDK, I came to the conclusion that I'd need to write a wrapper or ORM if I wanted my code to be expressive and in any way maintainable. Luckily, before I wasted a few days on a half-baked solution, I discovered Fireorm and it solved my troubles in *minutes*.

So thanks again Willy et al. If you're ever in SF hit me up, beer's on me! 🍻

**Comment by wovalle** [comment]
Thank you for your kind words @j1mmie ❤️
